delivery.yaml

```diff
@@ -52,6 +52,8 @@ pipeline:
           env:
           - name: E2E_NODE_OS
             value: ubuntu
+          - name: E2E_SKIP_CLUSTER_UPDATE
```

Comment:
Why?

Reply:
Just to ensure e2e is running against the new config.